pkgs/top-level/python-packages.nix

```diff
@@ -1585,6 +1585,13 @@ in {
 
   vdf = callPackage ../development/python-modules/vdf { };
 
+  vdirsyncer = disabledIf (!isPy3k) callPackage ../tools/misc/vdirsyncer {
+    inherit (pkgs.darwin.apple_sdk.frameworks) CoreServices Security;
+    inherit (pkgs) pkg-config openssl rustPlatform;
+  };
+
+  vdirsyncerStable = disabledIf (!isPy3k) callPackage ../tools/misc/vdirsyncer/stable.nix { };
```

Comment:
only *libraries* should be imported, not applications. Either move `vdirsyncer` under `python-modules` and make it use `buildPythonPackage`, or use `toPythonModule`. But please avoid that with pure Python packages because it would not make much sense. Here I argue moving `vdirsyncer` is the preferred solution.

Reply:
I'm not sure I understand what you mean "moving" because it's already in python-packages.nix, but I agree that using `buildPythonPackage` is more appropriate now.

Reply:
> Either move vdirsyncer under `python-modules`

`python-modules` is the folder, so this is about the the actual expression.

Reply:
I see, done.